protected/humhub/modules/admin/resources/js/humhub.admin.pending.registrations.js

```diff
@@ -0,0 +1,51 @@
+humhub.module('admin.pending.registrations', function (module, require, $) {
+    module.initOnPjaxLoad = true;
+
+    function hasChecked($checkBoxes) {
+        var result = false;
+        $checkBoxes.each(function () {
+            if ($(this).prop("checked")) {
+                result = true;
+                return false;
+            }
+        });
+        return result;
+    }
+
+    var init = function (isPjax) {
+
+        $("input").change(function () {
+            var $checkBoxes = $('.regular-checkbox');
+
+            if (hasChecked($checkBoxes)) {
+                $('.btn-delete').html('Delete selected rows')
+            } else {
+                $('.btn-delete').html('Delete all')
+            }
+        });
+
+        $(".btn-delete").on("click", function (e) {
+            var keys = $("#grid").yiiGridView("getSelectedRows");
+            if (keys.length > 0) {
+                e.preventDefault();
+                if(confirm("Delete selected invitations?")){
+                    $.ajax({
+                        url: "/index.php?r=admin%2Fpending-registrations%2Fdelete-all",
```

Comment:
Do not use static urls in javascript, this will fail for example when using url rewriting rules. Check https://docs.humhub.org/docs/develop/javascript-index#module-configuration for how to transfer such data to js as module configuration or when using a JsWidget, you can use the widget options which are provided as `data-*` attributes on the widget root.

Reply:
Done with Widget!